Turner.Infrastructure.Mediator.EF/Configuration/SimpleInjectorMediatorConfiguration.cs

```diff
@@ -0,0 +1,15 @@
+﻿using SimpleInjector;
+using System.Reflection;
+using Turner.Infrastructure.Mediator.Decorators;
+
+namespace Turner.Infrastructure.Mediator.Configuration.EF
+{
+    public static class SimpleInjectorMediatorConfiguration
+    {
+        private static void RegisterHandlers(Container container, Assembly[] assemblies)
+        {
```

Comment:
Consider renaming this function to make it clear that its job is to register the transaction decorator.  Also consider making it an extension method on `Container` to make it consistent with the mediator configuration methods.